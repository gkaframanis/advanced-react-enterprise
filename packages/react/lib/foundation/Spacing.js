const spaces = {
    none: 'none',
    xxxs: 'xxxs',
    xxs: 'xxs',
    xs: 'xs',
    sm: 'sm',
    md: 'md',
    lg: 'lg',
    xl: 'xl',
    xxl: 'xxl',
    xxxl: 'xxxl',
};
var Spacing = Object.freeze(spaces);

export { Spacing as default };
//# sourceMappingURL=Spacing.js.map
