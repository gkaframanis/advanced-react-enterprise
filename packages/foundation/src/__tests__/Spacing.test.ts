import Spacing from '../Spacing'

test('snapshot of spacing', () => {
    expect(Spacing).toMatchSnapshot()
})
