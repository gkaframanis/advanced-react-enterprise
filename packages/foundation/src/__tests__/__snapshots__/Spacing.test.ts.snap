// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of spacing 1`] = `
{
  "lg": "lg",
  "md": "md",
  "none": "none",
  "sm": "sm",
  "xl": "xl",
  "xs": "xs",
  "xxl": "xxl",
  "xxs": "xxs",
  "xxxl": "xxxl",
  "xxxs": "xxxs",
}
`;
